src/main/java/li/cil/tis3d/api/ClientAPI.java

```diff
@@ -0,0 +1,14 @@
+package li.cil.tis3d.api;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+@Environment(EnvType.CLIENT)
+public final class ClientAPI {
+    public enum Font {
+        SmallFont,
+        NormalFont
+    }
```

Comment:
Not sure if this enum should just wholesale move into `FontRendererAPI` now.

Reply:
I think that would make sense, yeah!